### ISSUE TITLE
How to run an application built in the latest build under Ubuntu 14.04

### ISSUE THREAD

**Issue by ivanscm** [electron-userland/electron-builder#6434]
```
/opt/Complex/resources/app/node_modules/bindings/bindings.js:121 Uncaught Error: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /opt/Complex/resources/app/node_modules/@serialport/bindings/build/Release/bindings.node)
    at process.func [as dlopen] (electron/js2c/asar_bundle.js:5)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:1138)
    at Object.func [as .node] (electron/js2c/asar_bundle.js:5)
    at Module.load (internal/modules/cjs/loader.js:935)
    at Module._load (internal/modules/cjs/loader.js:776)
    at Function.f._load (electron/js2c/asar_bundle.js:5)
    at Function.o._load (electron/js2c/renderer_init.js:33)
    at Module.require (internal/modules/cjs/loader.js:959)
    at require (internal/modules/cjs/helpers.js:88)
    at bindings (/opt/Complex/resources/app/node_modules/bindings/bindings.js:112)
```

**Comment by neoxpert** [comment]
Did you build and package the application by yourself? The problem seems not to be directly related to electron builder but non satisfied dependencies. I don't know if the required glibc version is available for Ubuntu 14.04, but that's a required dependency of the @serialport module. Regarding the version 2.28 there has been an an issue within that project https://github.com/serialport/node-serialport/issues/2301 .

**Comment by Lyttell** [comment]
> ```
> /opt/Complex/resources/app/node_modules/bindings/bindings.js:121 Uncaught Error: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /opt/Complex/resources/app/node_modules/@serialport/bindings/build/Release/bindings.node)
>     at process.func [as dlopen] (electron/js2c/asar_bundle.js:5)
>     at Object.Module._extensions..node (internal/modules/cjs/loader.js:1138)
>     at Object.func [as .node] (electron/js2c/asar_bundle.js:5)
>     at Module.load (internal/modules/cjs/loader.js:935)
>     at Module._load (internal/modules/cjs/loader.js:776)
>     at Function.f._load (electron/js2c/asar_bundle.js:5)
>     at Function.o._load (electron/js2c/renderer_init.js:33)
>     at Module.require (internal/modules/cjs/loader.js:959)
>     at require (internal/modules/cjs/helpers.js:88)
>     at bindings (/opt/Complex/resources/app/node_modules/bindings/bindings.js:112)
> ```

TUSC/obj/Debug/TemporaryGeneratedFile_036C0B5B-1481-4323-8D20-8F5ADCB23D92.cs
